### DISCUSSION TITLE
Ton SDK for Godot

### DISCUSSION THREAD

**Discussion by Ruonia** [ton-org/game-engines-sdk#10] [Q&A] [answered]
Hi, I love building games in godot and now decided to experiment with Web3 features in my game, token claims, nft minting and so on. 
I'm acknowledged that there is a Solana Game SDK for godot managed by the community, but is there anything like this for Ton ? (Cause i like it more lol).   

I've found an sdk only for phaser & js . If there is none, how can i implement web3 functions in my godot game? Where should i start?

**Comment by barinbritva** [comment, marked as answer]
@Ruonia hi,

I'm glad you are interested in Games & TON development

Unfortunately, as I can see there is no implementation for Godot. This repo is all about JS projects.

I saw a few implementations for Unity. Godot also uses C#, so probably you could find some C# examples. And copy C# code from Unity libraries.

Here what's I found for Unity:
- https://github.com/continuation-team/unity-ton-connect
- https://github.com/MrVeit/Veittech-UnitonConnect

I hope it helps a bit
